packages/typespec-azure-resource-manager/lib/common-types/managed-identity.tsp

```diff
@@ -32,9 +33,14 @@ model ManagedIdentityProperties {
   type: ManagedServiceIdentityType;
 
   @doc("The identities assigned to this resource by the user.")
-  userAssignedIdentities?: Record<UserAssignedIdentity>;
+  userAssignedIdentities?: UserAssignedIdentities;
 }
 
+/** The set of user assigned identities associated with the resource. The userAssignedIdentities dictionary keys will be ARM resource ids in the form: '/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ManagedIdentity/userAssignedIdentities/{identityName}. The dictionary values can be empty objects ({}) in requests.", */
+model UserAssignedIdentities is Record<UserAssignedIdentity>;
```

Comment:
is this named recored necessary?

Reply:
It could cause SDK generation difference. Though I do not have direct usage data on this common type, I am playing it safe to match the common-type definition.

Reply:
I see there is a definition called `UserAssignedIdentities` in the common types